src/megatron/bridge/training/initialize.py

```diff
@@ -154,6 +154,7 @@ def finish_mpu_init():
             rng_config.data_parallel_random_init,
             rng_config.te_rng_tracker,
             rng_config.inference_rng_tracker,
+            use_cudagraphable_rng=model_config.enable_cuda_graph or model_config.external_cuda_graph,
```

Comment:
could we also need these checks in the config container validate?

https://github.com/NVIDIA/Megatron-LM/blob/5653514e7c70a2adec71935d20c01ded5508c8b3/megatron/training/arguments.py#L1149-L1166